### PR TITLE
Update godep for kubelet runtime API

### PR DESCRIPTION
**Pull request by feiskyer**
This PR: 

* 	Update godeps for kubelet runtime API
* 	Add pkg/hyper/api for hyperd gRPC API

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/frakti/15)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+sudo: required
 
 go:
   - 1.6.3
 go_import_path: k8s.io/frakti
 
 install:
   - go install -v $(go list ./... | grep -v /vendor/)
-  - ./hack/verify-all.sh
+  - hack/install-protoc.sh
```
Comment (resouer) [inline .travis.yml]: ./hack, inconsistency

Comment (feiskyer) [inline .travis.yml]: Fixed.

File: hack/install-protoc.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+which yum>/dev/null
+if [[ $? != 0 ]]; then
+  sudo apt-get install -y unzip
+else
+  sudo yum install -y unzip
+fi
+
+# Install protoc
+cd /tmp
+curl -sSL https://github.com/google/protobuf/releases/download/v3.0.0-beta-3/protoc-3.0.0-beta-3-linux-x86_64.zip -o protoc-3.0.0-beta-3-linux-x86_64.zip
```
Comment (resouer) [inline hack/install-protoc.sh]: It will be better to have ${protoc_version} here, since we are using a beta and it is tend to be changed.

**Comment by resouer** [comment]
some nits in scripts, others lgtm

File: hack/install-protoc.sh (hunk 2 above)
Comment (feiskyer) [inline hack/install-protoc.sh]: It's my mistake, forgot to remove this line. It should install protoc-v3.0.0.

**Comment by feiskyer** [comment]
@resouer Addressed comments and rebased. PTAL.

File: hack/verify-generated-hyperd-api.sh [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+ret=0
+hack/update-generated-hyperd-api.sh
+diff -I "gzipped FileDescriptorProto" -I "0x" -Naupr ${_tmp}/api.pb.go ${PROTO_ROOT}/api.pb.go || ret=$?
+if [[ $ret -eq 0 ]]; then
+    echo "Generated hyperd api from proto up to date."
+else
+    echo "Generated hyperd api from proto is out of date. Please run hack/update-generated-hyperd-api.sh"
+    exit 1
+fi
```
Comment (resouer) [inline hack/verify-generated-hyperd-api.sh]: blank line

Comment (feiskyer) [inline hack/verify-generated-hyperd-api.sh]: Removed.

**Comment by resouer** [comment]
lgtm

**Pull request merged**